### ISSUE TITLE
Inkrato API node module

### ISSUE THREAD

**Issue by ISNIT0** [Inkrato/inkrato#2]
I'm happy to make a wrapper for the Inkrato http API if you think it is worth the time?

**Comment by iaincollins** [comment]
That's a great idea! I'd happily fix up the API too if you have any recommendations (I'm aware there are things missing I'd like to add). - I'd love to chat to you about this over coffee if you are free some time!

**Comment by ISNIT0** [comment]
Absolutely, I'm fairly busy for the next few weeks, but it'll be good to catch up! I'll send an e-mail when I'm free!
